rootfs-builder/centos/Dockerfile

```diff
@@ -1 +1,7 @@
 FROM centos:7
+
+ADD https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz /tmp
```

Comment:
This looks fine, but we're hard-coding more into these `Dockerfile`'s which is going to make them more difficult to maintain.

I'd rather we fix #31 first (maybe as one of the commits on this PR) to avoid this. These 2 chunks of code are identical too which lends weight to doing that as you could add something like:

```
FROM centos:@centos_version@

@centos_install_go@
```

And,

```
from euleros:@euleros_version@

# EulerOS is similar enough to CentOS to re-use this code.
@centos_install_go@
```

Lastly, there is the issue of the version of golang. The CI systems for the various code repos are using go `1.8.3` so we're in danger of repeating https://github.com/clearcontainers/packaging/issues/257.

Could you create a top-level `./versions.txt` containing a `go_version=` so that it's defined in a single location? It maybe be best to default to `1.8.3` for now and raise an issue to discuss go version support separately.

Reply:
that is a good point, let me add  Dockerfile templates.